app/models/media_attribute.rb

```diff
@@ -0,0 +1,34 @@
+# == Schema Information
+#
+# Table name: media_attribute
+#
+#  id            :integer          not null, primary key
+#  high_title    :string           not null
+#  low_title     :string           not null
+#  neutral_title :string           not null
+#  slug          :string           not null, indexed
+#  title         :string           not null, indexed
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+# Indexes
+#
+#  index_media_attribute_on_slug   (slug)
+#  index_media_attribute_on_title  (title)
+#
+
+class MediaAttribute < ActiveRecord::Base
+  self.table_name = 'media_attribute'
```

Comment:
rails complained about something when i used the resource generator with `MediaAttributes`, and somethings didn't work so to get the errors to go away I overridden the default